com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -178,6 +183,26 @@ internal set
         /// </summary>
         public bool DontDestroyWithOwner;
 
+        public InterestSettings InterestSettingsOverride;
+        public InterestSettings InterestSettings
```

Comment:
so we're saying that devs could override interest management per networkobject rather than creating an ad-hoc, specific interest management node plugged into networkmanager, right? is this the intention here or is this just temporary?

Reply:
Yeah, though you've helped me notice something I overlooked - yes, the idea was you can override the interest settings on a prefab (it's NetworkObject).  If you don't specify that, it's supposed to go to its Interest Node, which has replication settings, and if not there, to the NetworkManager default settings.  But because you can have > 1 InterestNodes, I now realize that creates an odd situation on which of those nodes to go off of.  Need to think about this more, thank you.

Beyond that wart though, the thinking here was like a character might normally have one set of settings (e.g. a player) but then like we want the ability say "if it's a player I just killed, show the corpse regardless even if out of my immediate range and I'd normally not see it")

Reply:
So I simplified it a bit; now the Interest Settings are just a property on both the NetworkManager (defaults) and the Prefab itself (no longer a property of an interest node).  

That is to say
- you can define a global setting on the NM
- you can define a setting on the prefab, if so, that wins
- you can stomp on the settings on an instance of a prefab, and if so, that setting wins

Reply:
Since these are all exposed as public fields they could be changed at runtime (while an object is spawned). Is this intended?